catalog_validation/items/questions_utils.py

```diff
@@ -151,6 +151,10 @@ def normalise_question(question: dict, version_data: dict, context: dict) -> Non
                 {'value': i['id'], 'description': f'{i["name"]!r} Certificate Authority'}
                 for i in context['certificate_authorities']
             ]
+        elif ref == 'definitions/port':
+            data['enum'] += [
```

Comment:
If the port can be `null` in it's schema, you should specify that in the enum as well as otherwise this must be specified with a specific port number always which might not always be appropriate if user wants to disable some services and the chart dev allows it.